aries-site/src/pages/templates/global-header.mdx

```diff
@@ -1,5 +1,6 @@
 import { Box, Button, Image, Notification } from 'grommet';
-import { AppsRounded, Figma, Language, Search, Shop } from 'grommet-icons';
+import { Apps, Language, Search, Shop } from '@hpe-design/icons-grommet';
```

Comment:
Need to update the references to the icons later in this file to use the new icon names

Reply:
updated